translation/en/RedirectedResources/assets/abdata/adv/scenario/c30/14/17/translation.txt

```diff
@@ -1,4 +1,4 @@
-「んーっ、よく寝た。……あれ？　メモがある。なになに――」=“Mmm, I slept well. ...Oh? There's a letter. What's it say...?”
-//『きちんと起きられたかしら？　お姉さんはお寝坊なキミが心配です』=
-//『本当は朝から色んなことをして、もっともっとラブラブな感じでいたかったけど……残念ながら今日は早く行かなきゃいけないの』=
-//『また後で……いっぱいラブラブしましょうね。ほんのちょっとでも離れたくないくらい……大好きよ♪　[H名]』=
+「んーっ、よく寝た。……あれ？　メモがある。なになに――」=Mmm, I slept well. Oh? There's a note, let's see ――
+『きちんと起きられたかしら？　お姉さんはお寝坊なキミが心配です』=~Did you wake up properly? Onee-san is worried about you, Mr. Sleepyhead.~
+『本当は朝から色んなことをして、もっともっとラブラブな感じでいたかったけど……残念ながら今日は早く行かなきゃいけないの』=~I really wanted to be there with you and do more lovey-dovey stuff right till noon, but unfortunately today I had to leave really early, sorry.~
+『また後で……いっぱいラブラブしましょうね。ほんのちょっとでも離れたくないくらい……大好きよ♪　[H名]』=~Later, let's do a lot of lovey-dovey stuff together, alright? I don't wanna be separated from you, even for a little bit... I love you ♪! -- From [H名]~
```

Comment:
The note lines (`adv/scenario/c*/*/17`) should be quoted with `"` rather than `~`

Reply:
already solved